### PR TITLE
Seat map fixes

### PR DESCRIPTION
**Pull request by igorp1**
__what__

This PR corrects the disable state of the seat map card.  

1. It should not show the hover state when the button is disabled. [Reported issue here](https://github.com/duffelhq/duffel-components/issues/214)
2. It should disable the card if there's no available services, even when the seat map is returned. [Reported issue here](https://github.com/duffelhq/duffel-components/issues/213)

### CODE REVIEW

File: src/styles/components/Card.css [hunk 1]
```diff
@@ -1,7 +1,8 @@
 .ancillary-card:disabled {
-  color: inherit;
-  cursor: not-allowed;
-  background-color: var(--GREY-100);
+  /* important needed to take prescedence over default button styles */
```
Comment (andrejak) [inline src/styles/components/Card.css:2]:
```suggestion
  /* important needed to take precedence over default button styles */
```

File: src/lib/hasAvailableSeatService.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { SeatMap } from "@duffel/api/types";
```
Comment (andrejak) [inline src/lib/hasAvailableSeatService.ts:1]: Could potentially add a quick test for this?

**Review by andrejak — APPROVED** [review]

File: src/styles/components/Card.css (hunk 1 above)
Comment (igorp1) [reply src/styles/components/Card.css:2]: Oops, thank you, fixed!

File: src/lib/hasAvailableSeatService.ts (hunk 2 above)
Comment (igorp1) [reply src/lib/hasAvailableSeatService.ts:1]: Good shout, added!

**Pull request merged**